CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.0.0
+
+This is a major release
+
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
+  Notably, this change drops Java 7 support as Jetty 9.3 and higher only
+  support Java 8 and higher. No public APIs have changed in this release.
+
+* The behavior of normalize-uri-path has changed slightly due to changes in
+  Jetty's URIUtil decodePath method. Specifically, overlong encodings change
+  their decoded value (but are still not traversable), and semicolons no longer
+  terminate URI processing, but only when ; is followed at some point by
+  another path segment beginning with /. For example, "/foo/bar;bar=chocolate/baz;baz=bim"
+  now decodes to "/foo/bar/baz" while before it decoded to "/foo/bar".
+  (https://github.com/eclipse/jetty.project/commit/7f62f2600b943b9aed0e4771891939bf61372c5a)
```

Comment:
Huh, weird.  Oh, well.  Probably good to add/update the blurb about this to the [request-uri-normalization](https://github.com/puppetlabs/trapperkeeper-webserver-jetty9#request-uri-normalization) section in the README.md doc.